assessments/2022-10-31-strings/code.js

```
@@ -10,3 +10,33 @@ const rand = (n) => Math.floor(Math.random() * n);
 // reasonable argument list. To get full credit you need to write a correct
 // function but you can get partial credit for a function that is basically
 // correct even if contains small mistakes.
+
+const firstHalf = (s) => {
+  return s.substring (0, (s.length / 2))
+}
+
+const secondHalf = (s) => {
+  return s.substring ((s.length / 2), s.length)
+}
+
+const upDown = (s) => {
+  return s.toUpperCase() + s.toLowerCase()
+}
+
+const firstFewEveryOther = (s) => {
+  if (s.length >= 5) {
```

As we talked about in office hours, when the function description says the argument will be something (five or more characters in this case) you can take that as a given and don't need to test it. (In general though, it's often good to put in checks like this for characteristics that an argument must have. But if they don't you need to do something more than just silently returning undefined which is what this function would do. We'll talk more about that kind of thing later in the year.)